### DISCUSSION TITLE
An option to control auto-close separately

### DISCUSSION THREAD

**Discussion by iafelix** [zed-industries/zed#23960] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Would be nice to have the `auto_close_<something>` for each entry as requested on #4543 each one of us has different needs/taste as such the #7612 that want to change just a single behavior (the brackets) and the setting `auto_close` will change all of them (parentheses, brackets, quotes, etc) in my case I would like to disable only the double quote as per requested on #4543 but keep the others.

**Comment by cole-miller** [comment]
Thanks for the idea---this seems like something we should talk through in the discussions.
